docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Advanced Components - Overview
+---
+
+# Advanced Components - Overview
+
+<p class="description">A collection of fast and powerful components.</p>
+
+## Licenses
+
+While MUI Core is entirely licensed under MIT, MUI X serves a part of its components under a commercial license.
+Please pay attention to the license.
+
+### Community Plan
+
+Published under [MIT license](https://tldrlegal.com/license/mit-license) and [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf)
+
+The packages of the **Community** Plan are:
+
+- [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
+- [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
+
+### Pro Plan
+
+This plan contains the features that are at the limit of what the open-source model can sustain. From a price perspective, the plan is designed to be accessible to most professionals.
+
+See [Pricing](https://mui.com/store/items/material-ui-pro/) for details on purchasing licenses.
```

Comment:
Maybe this page is more appropriate than the store to discover the pricing strategy
```suggestion
See [Pricing](https://mui.com/pricing/) for details on purchasing licenses.
```